### ISSUE TITLE
Missing space not detected in declaration assignment

### ISSUE THREAD

**Issue by jeremypw** [vala-lang/vala-lint#147]
The following code does not trigger a lint error:
```
int i =0;
```

There should be a space after the `=`.
